plugins/dell/fu-plugin-dell.c

```diff
@@ -286,6 +286,7 @@ fu_plugin_dock_node (FuPlugin *plugin, const gchar *platform,
 		dock_name = g_strdup_printf ("Dell %s", dock_type);
 	}
 	fu_device_set_vendor (dev, "Dell Inc.");
+	fu_device_set_vendor_id (dev, "TPM:DELL");
```

Comment:
Actually I think 0x1028 is fine to use here instead.
https://devicehunt.com/view/type/pci/vendor/1028

Reply:
I pondered PCI:0x1028 like the other Dell device, but this is logically a TPM device, and it's a vendor ID of DELL. I know DELL isn't a TPM vendor according to the spec, but in this case I think it's valid. That said, it's probably your decision which you want to use. :) What do you prefer?

Reply:
Well the thing that is vendor ID Dell is for representing the "Capsule".  The TPM is a Nuvoton or Infineon or ST Micro, etc.

Reply:
And "Dell" would be the one uploading said payload to LVFS not one of the TPM vendors.

Reply:
> is for representing the "Capsule".

That's a compelling argument. I'll switch it round.